### PR TITLE
chore: add Neetcode in Competitive Programming category

### LINKED ISSUE #1918: [Add] Neetcode in Competitive Programming Category
### Site URL

https://www.youtube.com/@NeetCode

### Description of the above provided site

This is a youtube channel which help students and professional to become better in Competitive Programming and Tech Interviews

### Category

Competitive Programming

### Sub Category

tutorials

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by singodiyashubham87**
## Fixes Issue
Fixes #1918 

## Changes proposed

I have added the Neetcode youtube channel into the database of the Competitive Programming category and validated the JSON too.

## Screenshots
image

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
@singodiyashubham87 please don't remove any existing resource, kindly add Tushar Roy back

**Comment by singodiyashubham87** [comment]
@rupali-codes I'm really sorry for the mistake, I have rectified my mistake, kindly check it once.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by Anmol-Baranwal** [comment]
@CBID2 
Please review.

File: database/competitive_programming/cp-tutorials.json [hunk 1]
```diff
@@ -46,5 +46,13 @@
     "category": "competitive-programming",
     "subcategory": "cp-tutorials",
     "language": "english"
+  },
+  {
+    "name": "Neetcode",
+    "description": "This is a youtuber helps students and professional to become better in Competitive Programming and Tech Interviews.",
```
Comment (CBID2) [inline database/competitive_programming/cp-tutorials.json:52]:
```suggestion
    "description": "This video helps students and professionals to become better in Competitive Programming and tech interviews.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @singodiyashubham87. I just gave some tips on how you can improve your entry.

File: database/competitive_programming/cp-tutorials.json (hunk 1 above)
Comment (singodiyashubham87) [reply database/competitive_programming/cp-tutorials.json]: @CBID2 I have updated the description

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — APPROVED** [review]
Looks great to me

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**